src/current/_includes/releases/v23.1/v23.1.25.md

```diff
@@ -0,0 +1,80 @@
+
+- Added the `sql.auth.grant_option_inheritance.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, the [`GRANT OPTION`]({% link v23.1/grant.md %}) is not inherited via role membership, which will prevent the descendant roles from being able to grant the privilege to others. The privilege itself is still inherited via role membership. [#126862][#126862]
+- Added the `sql.auth.grant_option_for_owner.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, then the [`GRANT OPTION`]({% link v23.1/grant.md %}) is not implcitly given to the owner of an object. The object owner still implicitly has all privileges on the object, just not the ability to grant them to other users. [#127005][#127005]
+
+<h3 id="v23-1-25-db-console-changes">DB Console changes</h3>
+
+- The [Database]({% link v23.1/ui-databases-page.md %}) details and Table details pages in the [DB Console]({% link v23.1/ui-overview.md %}) now display the correct `"stats last created"` value. [#126415][#126415]
+- The [Database]({% link v23.1/ui-databases-page.md %}) and Tables pages in the [DB Console]({% link v23.1/ui-overview.md %}) will show a loading state while loading information for databases and tables including size and range counts. [#127699][#127699]
+- In the [Database]({% link v23.1/ui-databases-page.md %}) details page, the table name will no longer appear with quotes around the schema and table name. [#127763][#127763]
+- [`ALTER ROLE`]({% link v23.1/alter-role.md %}) events in the [DB Console]({% link v23.1/ui-overview.md %}) event log now render correctly when the event does not contain any role options. [#126563][#126563]
+- Fixed a bug where the [Databases]({% link v24.1/ui-databases-page.md %}) page crashed if the range information was not available. [#127090][#127090]
+
+<h3 id="v23-1-25-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in which the [`DISCARD`]({% link v23.1/alter-table.md %}) statement was disallowed with [`default_transaction_read_only = on`]({% link v23.1/session-variables.md %}). [#127549][#127549]
+- Fixed a bug where [`ALTER DATABASE ... DROP REGION`]({% link v23.1/alter-database.md %}#drop-region) could fail if any tables under the given database have [indexes on expressions]({% link v23.1/expression-indexes.md %}). [#126600][#126600]
+- Fixed a bug when [restoring]({% link v23.1/restore.md %}) a database with a [composite type]({% link v23.1/create-type.md %}#create-a-composite-data-type). [#126847][#126847]
+- Fixed a bug when inputting `public` role as user name for [built-in compatibility functions]({% link v23.1/functions-and-operators.md %}#compatibility-functions), such as `has_database_privilege` and `has_schema_privilege`. [#126853][#126853]
```

Comment:
public role could link to https://www.cockroachlabs.com/docs/v23.1/security-reference/authorization.html#default-roles